pengwin-setup.d/docker.sh

```diff
@@ -18,13 +18,18 @@ function docker_install_build_relay() {
   if [[ ! -f "${wHome}/.npiperelay/npiperelay.exe" ]]; then
 
     echo "Checking for go"
-    if ! (go version); then
+    if ! command_check '/usr/local/go/bin/go' 'version' ; then
       echo "Downloading Go using wget."
       wget -c "https://dl.google.com/go/go${GOVERSION}.linux-$(dpkg --print-architecture).tar.gz"
       tar -xzf go*.tar.gz
 
       export GOROOT=$(pwd)/go
       export PATH="${GOROOT}/bin:$PATH"
+    else
+      # Whether it was just installed previously, or right now,
+      # makes sure to set correct env variables
+      export GOROOT=/usr/local/go
```

Comment:
What happens if Go was installed via **apt**? it will be in another path

Reply:
modified this in latest commit. there's now 3 different return values from the command_check so these variables are only set if it's a fresh go install performed via pengwin-setup